systest/tests/smeshing_test.go

```diff
@@ -31,7 +31,13 @@ func TestSmeshing(t *testing.T) {
 func testSmeshing(t *testing.T, tctx *testcontext.Context, cl *cluster.Cluster) {
 	const limit = 15
 
-	createdch := make(chan *spacemeshv1.Proposal, cl.Total()*limit)
+	first := currentLayer(tctx, t, cl.Client(0))
+	// TODO fetch epoch size from API
+	first = nextFirstLayer(first, 4)
```

Comment:
magic number?

Reply:
it is a number o layers in the epoch, need to add an api to fetch it from the node